dali/kernels/imgproc/paste/paste.h

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_PASTE_PASTE_H_
+#define DALI_KERNELS_IMGPROC_PASTE_PASTE_H_
+
+#include <utility>
+#include "dali/util/ocv.h"
+#include "dali/core/common.h"
+#include "dali/core/convert.h"
+#include "dali/core/geom/box.h"
+#include "dali/core/error_handling.h"
+#include "dali/kernels/kernel.h"
+#include "dali/kernels/imgproc/roi.h"
+
+#define X_AXIS 0
+#define Y_AXIS 1
+#define C_AXIS 2
+
+namespace dali {
+namespace kernels {
+
+template<typename OutputType, typename InputType>
+class PasteCpu {
+ public:
+  using Image = InTensorCPU<InputType, 3>;
+  using OutImage = OutTensorCPU<OutputType>;
```

Comment:
Why limit ourselves to 2D?
Why does the input have fixed dimensionality, but not the output?

Reply:
Limited to 2D for simplicity's sake.  Missing dimensionality in Out was a typo, fixed now.